mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -218,23 +232,30 @@ def remove_repeated_time_index(ds): #{{{
     # remove repeated indices
     ds = ds.isel(Time=index)
 
-    return ds #}}}
+    return ds  # }}}
 
-def test_load_mpas_xarray_datasets(path): #{{{
-    ds = xr.open_mfdataset(path, preprocess=lambda x: preprocess_mpas(x, yearoffset=1850))
+
+def test_load_mpas_xarray_datasets(path):  # {{{
+    ds = xr.open_mfdataset(path, preprocess=lambda x:
+                           preprocess_mpas(x, yearoffset=1850))
     ds = remove_repeated_time_index(ds)
 
     # make a simple plot from the data
     ds.Time.plot()
     plt.show()
 
-    return #}}}
+    return  # }}}
+
 
-def test_load_mpas_xarray_timeSeriesStats_datasets(path): #{{{
-    ds = xr.open_mfdataset(path, preprocess=lambda x: preprocess_mpas(x,
-        timeSeriesStats=True, timestr='timeSeriesStatsMonthly_avg_daysSinceStartOfSim_1'))
+def test_load_mpas_xarray_timeSeriesStats_datasets(path):  # {{{
+    timestr = 'timeSeriesStatsMonthly_avg_daysSinceStartOfSim_1'
+    ds = xr.open_mfdataset(path, preprocess=lambda x:
+                           preprocess_mpas(x,
+                                           timeSeriesStats=True,
+                                           timestr=timestr))
     ds = remove_repeated_time_index(ds)
-    ds2 = xr.open_mfdataset(path, preprocess=lambda x: preprocess_mpas(x, yearoffset=1850))
+    ds2 = xr.open_mfdataset(path, preprocess=lambda x:
+                            preprocess_mpas(x, yearoffset=1850))
```

Comment:
I agree, this is certainly a gray-area and the code is clear and aesthetically pleasing so we go for it and just keep and eye out for better ways to do this in the future (if they exist).